tensorrt_llm/_torch/auto_deploy/transform/library/mxfp4_moe.py

```
@@ -189,6 +189,17 @@ def _register_mxfp4_expert_params(
     experts_mod.register_parameter(dn_blocks_name, nn.Parameter(dn_blocks, requires_grad=False))
     experts_mod.register_parameter(dn_scales_name, nn.Parameter(dn_scales, requires_grad=False))
 
+    # Free the now-unused bf16 stacked weight params (`gate_up_proj`, `down_proj`).
+    # The biases (`gate_up_proj_bias`, `down_proj_bias`) are still consumed by
+    # ``triton_mxfp4_moe`` and must remain. For models like GPT-OSS-120B
+    # (128 experts × 36 layers × ~33 MB per layer of bf16 placeholder) freeing
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)